assets/r/evaluate_model_rpart.R

```diff
@@ -61,10 +61,19 @@ pred_tr <- predict(model, newdata=trds, type="class")
 pred_tu <- predict(model, newdata=tuds, type="class")
 pred_te <- predict(model, newdata=teds, type="class")
 
+# 20241224 zy Save the predicted values based on full dataset.
+
+pred_tf <- predict(model, newdata=tcds, type="class")
```

Comment:
Looking good. Thanks. To be consistent it might be good to call it `tc` rather than `tf` internally in the code since it is based on tcds? And so perhaps better to call it `Complete` rather than `Full`.

Reply:
Done.